main.css

```diff
@@ -1,370 +0,0 @@
-#quote-box {
-  background-image: url("Images/parralax.jpg");
-  height: 200px;
-  background-attachment: fixed;
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: cover;
-  margin-bottom: 50px;
-}
-#center-quote {
-  vertical-align: middle;
-  text-align: center;
-  line-height: 200px;
-  font-style: italic;
-}
-#quote-text {
-    position: absolute;
-    z-index: 999;
-}
-/* - - - quote box end - - - - */ 
-
-/* - - - main content - - - - */ 
-#main {
-  width: 75%;
-  display: grid;
-  grid-template-columns: 2fr 1fr;
-  grid-template-rows: 1fr;
-  grid-template-areas: 
-    "left-content right-content";
-  margin-left: auto;
-  margin-right: auto;
-  grid-gap: 20px;
-}
-#left-content{
-  grid-area: left-content;
-}
-#aliens {
-  grid-area: aliens; 
-  position: relative;
-  justify-content: space-around;
-  align-items: flex-start;
-  display: flex;
-  flex-direction: column;
-  height: 50%;
-}
-.content-image {
-  height: 300px;
-  width: 100%;
-}
-#awesome-demo {
-  grid-area: awesome-demo;
-  grid-area: aliens; 
-  position: relative;
-  justify-content: space-around;
-  align-items: flex-start;
-  display: flex;
-  flex-direction: column;
-  height: 50%;
-}
-#right-content{
-  grid-area: right-content;
-  display: grid;
-  grid-template-rows: 1fr 1fr;
-  grid-template-columns: 1fr;
-  grid-template-areas: 
-  "main-nav-right"
-  "of-note";
-}
-.post-content {
-  border-bottom: solid grey 1px;
-  margin-bottom: 30px;
-  padding-bottom: 30px;
-}
-#main-nav-right {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-around;
-}
-.post{
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-}
-.post-head{
-  padding-top: 20px;
-  padding-bottom: 20px;
-}
-#note-chunk {
-  display: flex;
-}
-#of-note {
-  display: flex;
-  flex-direction: column;
-  }
-.note-text {
-  margin-bottom: 15px;
-}
-#note-box{
-  background-color: blueviolet;
-  width: 80px;
-  height: 80px;
-  float: left;
-  margin-right: 15px;
-  margin-bottom: 15px;
-}
-
-/*----"Learn More" CSS----*/
-
-#learn-main {
-  width: 75%;
-  display: grid;
-  grid-template-columns: 1fr 2fr;
-  grid-template-rows: 1fr;
-  grid-template-areas: 
-    "right-content left-content";
-  margin-left: auto;
-  margin-right: auto;
-  grid-gap: 20px;
-}
-/*----Post-Only CSS----*/
-#main-only{
-  display: flex;
-  justify-content: center;
-}
-#content-only{
-  width: 75%;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
-#head-only{
-  padding-bottom: 30px;
-}
-#pic-only{
-  padding-bottom: 30px;
-}
-/* - - - main content end - - - - */ 
-
-/*  - - - - footer  - - - - */
-#footer {
-  background-color: #d2d2d4;
-  position: static;
-  width: auto;
-  height: 350px;
-  margin-top: 30px;
-}
-#get-in-touch {
-  text-align: center;
-  padding-top: 10px;
-}
-#grid-for-footer {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-template-rows: minmax(275px) 1fr 1fr 1fr 1fr;
```

Comment:
take a look at this minmax usage here. I don't think it is working how you expect it to. There seems to be some layout issues with the grid in your footer. The minmax function takes two arguments, one being the minimum size it can be, and the second being the maximum size it can be. The way it is currently written, you are telling your grid to make 2 columns (which I believe you want) and 5 rows (which i don't think is what you want). The top row is trying to use minmax() to set the size, and then the four 1fr's following will split up the remaining space into 4 equally sized rows. Check out the documentation [here](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax). You also forgot to give your #form a grid-area value of 'form'. This could use a little reworking.